Removes some old vending machines icons, adds new ones

TobyThorne opened the pull request:
Очередной допил спрайтов автоматов.
- Чутка твикнул спрайты автоматов с сигаретами и кофе.
- Изменил цвета автомата с едой, теперь цвет не такой поносный (ичсх, буквально).
- Автомат со "здоровой" едой (в медбее) отрисован под новый формат.
- Автомат мага отрисован под новый формат.
- Изменён generic-автомат.
- Чутка доработана deny-анимация медицинского автомата.
- EngiVend теперь выглядит инженернее (оранжевее).
- Перерисовал театральный автомат (который с помадой и зеркальцами) под новый формат;
- Добавил алтернативые варианты спрайтов (выбираются рандомно) для кофейного, сигаретного и СБшного автоматов;
- Открученные от пола автоматы теперь падают на бок;
- Попутно пофиксил возможность покупать сразу несколько предметов, закликивая карточкой. Хе-хе.
![image](https://user-images.githubusercontent.com/45202681/97512602-1ca03080-19b4-11eb-894e-24b5f4bc6aba.png)
![image](https://user-images.githubusercontent.com/45202681/97512714-6e48bb00-19b4-11eb-9309-44b3d3528140.png)
![image](https://user-images.githubusercontent.com/45202681/97512725-74d73280-19b4-11eb-827a-24f18431984f.png)
![image](https://user-images.githubusercontent.com/45202681/97512733-7acd1380-19b4-11eb-993c-b5d5d79b6857.png)
![image](https://user-images.githubusercontent.com/45202681/97645743-f17f1500-1a77-11eb-81b6-1901d4b2a68d.png)
![theatervend](https://user-images.githubusercontent.com/45202681/97645840-35721a00-1a78-11eb-82ff-23baf122e8b8.gif)
![image](https://user-images.githubusercontent.com/45202681/97647668-0dd18080-1a7d-11eb-95b3-f25d2a96de51.png)

- [x] Pull Request полностью завершен, мне не нужна помощь чтобы его закончить.
- [x] Я внимательно прочитал все свои изменения и багов в них не нашел.
- [x] Я запускал сервер со своими изменениями локально и все протестировал.
- [x] Я ознакомился c [Guide to Contribute](https://github.com/ChaoticOnyx/OnyxBay/blob/dev/docs/contributing.md).

Commit:
vending machines

sprites tweaks

kreeperHLC commented:
![image](https://user-images.githubusercontent.com/14943437/97561774-b63a0300-19f1-11eb-920f-72f6e2eee012.png)
А эти пиксели - норма?

TobyThorne commented:
> ![image](https://user-images.githubusercontent.com/14943437/97561774-b63a0300-19f1-11eb-920f-72f6e2eee012.png)
> А эти пиксели - норма?

чё за магия, откуда они высрались

Commit:
vendors

- Fixed the ability to purchase multiple thingies at once;
- Added alternative sprites (picked randomly) for coffee, cig, and security vendors;
- Tweaked theatre vendors' icons;

TobyThorne commented:
- Перерисовал театральный автомат (который с помадой и зеркальцами) под новый формат;
- Добавил алтернативые варианты спрайтов (выбираются рандомно) для кофейного, сигаретного и СБшного автоматов;
- Попутно пофиксил возможность покупать сразу несколько предметов, закликивая карточкой. Хе-хе.

Commit:
drop the vend

Commit:
Update vending.dmi

- Added idle animations for alternative cigs and coffee vendors;

Commit:
Update vending.dmi

jammer312 commented on code/game/machinery/vending.dm:
```diff
@@ -223,8 +230,9 @@
 	else if((obj_flags & OBJ_FLAG_ANCHORABLE) && isWrench(W))
-		wrench_floor_bolts(user)
-		power_change()
+		if(wrench_floor_bolts(user))
```
По-моему оно так не работает (работает, но не так, как ты предполагаешь)
